### DISCUSSION TITLE
Unable to login -

### DISCUSSION THREAD

**Discussion by alec-prime** [mattdodge/yahoofantasy#70] [Q&A]
When I try to login into the developer app, the web browser is unable to reach the address and the terminal outputs the following message. 

> Launching OAuth handler server on localhost:8000
> Using localhost SSL certificate (HTTPS)
> Traceback (most recent call last):
>   File "/Library/Frameworks/Python.framework/Versions/3.12/bin/yahoofantasy", line 8, in <module>
>     sys.exit(yahoofantasy())
>              ^^^^^^^^^^^^^^
>   File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/click/core.py", line 1157, in __call__
>     return self.main(*args, **kwargs)
>            ^^^^^^^^^^^^^^^^^^^^^^^^^^
>   File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/click/core.py", line 1078, in main
>     rv = self.invoke(ctx)
>          ^^^^^^^^^^^^^^^^
>   File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/click/core.py", line 1688, in invoke
>     return _process_result(sub_ctx.command.invoke(sub_ctx))
>                            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>   File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/click/core.py", line 1434, in invoke
>     return ctx.invoke(self.callback, **ctx.params)
>            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>   File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/click/core.py", line 783, in invoke
>     return __callback(*args, **kwargs)
>            ^^^^^^^^^^^^^^^^^^^^^^^^^^^
>   File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/yahoofantasy/cli/login.py", line 68, in login
>     server.socket = ssl.wrap_socket(
>                     ^^^^^^^^^^^^^^^
> AttributeError: module 'ssl' has no attribute 'wrap_socket'

Any ideas or suggestions to resolve the error? Hoping it's something simple I've missed.

**Comment by mattdodge** [comment]
Looks like this changed in Python 3.12. I can work on getting that compatibility in place but in the meantime you'll probably have better luck using an earlier version of Python and running the CLI there

**Reply by drewffxi** [reply]
Hey mattdodge, was wondering if you ever got around to fixing this or if my best workaround is still installing old python

**Reply by mattdodge** [reply]
Haven't had a chance to look at this closer yet, will play around with some different python versions soon and see what's up
